src/status_im/ios/platform.cljs

```diff
@@ -114,4 +42,5 @@
    :group-block-shadows?         false
    :discover                     {:uppercase-subtitles? true}
    :public-group-icon-container  {:margin-top 2}
-   :private-group-icon-container {:margin-top 2}})
+   :private-group-icon-container {:margin-top 2}
```

Comment:
Can those 2 ones be moved to `styles` ?

Reply:
Oh, it turns out both are unused, so I just removed them.